### DISCUSSION TITLE
RW version must equal Vivado version?

### DISCUSSION THREAD

**Discussion by simux** [Xilinx/RapidWright#399] [General]
Hi RW maintainer,

I'm new to RW, my question is:  Can I use the latest RW with old vivado version, like RW 2021.2 with Vivado 2019.1?

I really don't want install so many vivado on my machine, :)

Regards
Joseph

**Comment by clavin-xlnx** [comment]
Hi Joseph,

Yes, the DCPs that are generated from the latest RapidWright should work with Vivado 2019.1.  Obviously there will be devices available in the latest RapidWright that won't be available in Vivado 2019.1, but I don't imagine that is what you are after.

Chris

**Comment by simux** [comment]
Hi Clavin,

Your answer save a lot of time for me. I will give it a try.
Thank you for your quick response. 

Regards
Joseph
